code/modules/mob/living/carbon/human/species_types/synths.dm

```diff
@@ -27,10 +26,13 @@
 	id = "military_synth"
 	armor = 25
 	punchdamage = 14
-	disguise_fail_health = 50
+	inherent_traits = list(TRAIT_NODISMEMBER,TRAIT_NOLIMBDISABLE,TRAIT_NOHUNGER,TRAIT_NOBREATH, TRAIT_NOHUNGER, TRAIT_TOXIMMUNE, TRAIT_DOOR_PRYER, TRAIT_NOSTAMCRIT, TRAIT_STRONG_GRABBER)
+	disguise_fail_health = 50 //This literally does nothing. This doesnt work.
```

Comment:
if this does nothing, remove it

Reply:
It technically has a use, but its major intended use was never implemented, and I might get around to it down the line